src/metabase/public_settings.clj

```diff
@@ -211,7 +211,7 @@
   (deferred-tru "Allow persisting models into the source database.")
   :type       :boolean
   :default    false
-  :visibility :admin)
+  :visibility :authenticated)
```

Comment:
@snoe wanted to check in with you that exposing `persisted-models-enabled` to authenticated users instead of admins is OK here. A user that only has "manage db" permission is not an admin and we need `persisted-models-enabled` value to figure out if we should show the "enable/disable" button at all

Reply:
Should we make it available only for admins or those who have a manage db permission for any db?

Reply:
@kulyk I have the same question here, does this make it possible for any auth user to toggle it on? Unfortunately, this one is not read-only.

Reply:
@snoe can I ask for your help with that? I'm not that productive with Clojure and our backend code, so I'm worried it would take too long for me to get that right

Reply:
@kulyk I checked with @dpsutton and since our `/persist/enable` `/persiste/disable` routes are properly locked down we are ok to make this change.

Reply:
Thanks @snoe!